code/modules/antagonists/eldritch_cult/eldritch_magic.dm

```diff
@@ -80,6 +80,33 @@
 	if(use_charge)
 		return ..()
 
+/obj/item/melee/touch_attack/mansus_fist/suicide_act(mob/user)
+	. = FIRELOSS
+	user.visible_message(span_suicide("[user] covers [user.p_their()] face with [user.p_their()] sickly-looking hand! It looks like [user.p_theyre()] trying to commit suicide!"))
+	var/mob/living/carbon/C = user	//iscarbon already used in spell's parent
```

Comment:
https://github.com/tgstation/tgstation/blob/master/.github/guides/STYLE.md#use-descriptive-and-obvious-names